certification/internal/operatorsdk/operatorsdk.go

```diff
@@ -163,9 +163,13 @@ func (o operatorSdk) BundleValidate(ctx context.Context, image string, opts Oper
 	return &bundleValidateData, nil
 }
 
-func (o operatorSdk) writeScorecardFile(resultFile, stdout string) error {
-	_, err := artifacts.WriteFile(resultFile, strings.NewReader(stdout))
-	return err
+func (o operatorSdk) writeScorecardFile(ctx context.Context, resultFile, stdout string) error {
```

Comment:
`o` is unused here.

Reply:
Think we should move this as a function vs. a method, or something else?

Reply:
Probably just a normal function.

Reply:
If we move this to a normal function, I'd probably push it down past `createScorecardConfigFile` just for clarity and to avoid confusion as to why all the other functions have `o` and this does not.